src/app/components/EditorInterface.tsx

```diff
@@ -204,8 +207,26 @@ const EditorInterface = () => {
     appState?.setEditorColor(
       (appState?.appState.settings as ISettings).backgroundColor,
     )
+    appState?.setBannerImgUrl(
+      (appState?.appState.settings as ISettings).bannerImage.url,
+    )
   }, [appState?.appState.settings])
 
+  useEffect(() => {
+    ;(async () => {
+      const imagePickerUtils = new ImagePickerUtils()
+      if (appState?.appState.bannerImgUrl instanceof Blob) {
+        setBannerImage(
+          (await imagePickerUtils.convertBlobToUrlString(
+            appState?.appState.bannerImgUrl,
+          )) as string,
+        )
+      } else {
+        setBannerImage(appState?.appState.bannerImgUrl as string)
+      }
+    })()
+  }, [appState?.appState.bannerImgUrl])
```

Comment:
this variable is in two useEffect dependency array, can we put the login in single useEffect @aatbip ?